### ISSUE TITLE
`IMAGE_DIFF_DIR` directory is empty after failing snapshot tests

### ISSUE THREAD

**Issue by fabiomassimo** [ashfurrow/Nimble-Snapshots#125]
As documented by `FBSnapshotTestCase` I have added the `IMAGE_DIFF_DIR` env variable to my Unit Test schema to save the failure diff images in a specific path in the `$(SOURCE_ROOT)`.

Although I have proper reference images and my snapshot tests are failing the folder at the path specified in the `IMAGE_DIFF_DIR` env variable still is empty.

How can I debug this issue better?

**Comment by ashfurrow** [comment]
Very strange, I'm not sure. I would double-check the set up, and check out the FBSnapshotTestCase for related issues: https://github.com/facebook/ios-snapshot-test-case It could be a bug on their end, or ours, maybe try reproducing the error without using Nimble-Snapshots to isolate the problem? Let us know 👍

**Comment by fabiomassimo** [comment]
After a reboot of my MacBook the issue is not reproducible anymore.

Thank you for your time and the tips for debugging. I'll make a note for any upcoming one, if any.

**Comment by ashfurrow** [comment]
Awesome, glad it's working! If you run into other problems or have any questions, please feel free to open an issue!

**Comment by fabiomassimo** [comment]
Just a quick follow up for whom might have this issue and the clue-less "power-off-power-on" trick does not work.

It actually happened that, after I have recorded the snapshots, I changed the specs description a little bit because of a typo. 
This caused to make a new run of Nimble-Snapshot (with recording mode off) to look up for snapshots that are not available because the name of the reference image is built on the description of your specs (which in my case were different when I recorded and when I test them).

The way I found this out was to look at the Xcode tests logs that are printed in the Xcode console and found out that no reference image was found.

Hope this will help and sorry for the silly mistake.
